weblate/static/editor/full.js

```diff
@@ -210,6 +236,20 @@
     });
   };
 
+  FullEditor.prototype.removeTranslationEntry = function (delete_url) {
+    $.ajax({
+      type: "DELETE",
+      url: delete_url,
+      headers: { "X-CSRFToken": this.csrfToken },
```

Comment:
The token should not be needed for the API.

```suggestion
```

Reply:
I removed it and getting 403. I think it;s needed!